Meadowlark-js/backends/meadowlark-postgresql-backend/src/repository/Get.ts

```diff
@@ -4,13 +4,15 @@
 // See the LICENSE and NOTICES files in the project root for more information.
 
 import type { PoolClient, QueryResult } from 'pg';
-import { GetResult, GetRequest } from '@edfi/meadowlark-core';
+import { GetResult, GetRequest, MeadowlarkId } from '@edfi/meadowlark-core';
 import { Logger } from '@edfi/meadowlark-utilities';
 import { findDocumentByIdSql } from './SqlHelper';
 
 const moduleName = 'postgresql.repository.Get';
 
-export async function getDocumentById({ meadowlarkId, traceId }: GetRequest, client: PoolClient): Promise<GetResult> {
+export async function getDocumentById({ documentUuid, traceId }: GetRequest, client: PoolClient): Promise<GetResult> {
+  // TODO *** cast to unknown is an invalid mixing of meadowlarkId and documentUuid
+  const meadowlarkId: MeadowlarkId = documentUuid as unknown as MeadowlarkId;
```

Comment:
This is wrong, but we aren't worrying about PostgreSQL right now. The nice thing is that a future search for "as unknown as" will find all the problem areas.